etl/steps/data/grapher/who/2023-04-03/flu_elderly.py

```diff
@@ -0,0 +1,37 @@
+"""Load a garden dataset and create a grapher dataset."""
```

Comment:
Maybe a little more context for these steps can help (I would like `walkthrough` did this anyway)